src/js/utils/browser.js

```diff
@@ -212,3 +173,42 @@ export const TOUCH_ENABLED = Dom.isReal() && (
   'ontouchstart' in window ||
   window.navigator.maxTouchPoints ||
   window.DocumentTouch && window.document instanceof window.DocumentTouch);
+
+/**
+ * Whether or not this device is an iPad.
+ *
+ * @static
+ * @const
+ * @type {Boolean}
+ */
+export const IS_IPAD = (/iPad/i).test(USER_AGENT) || (IS_SAFARI && TOUCH_ENABLED);
```

Comment:
Could it be that `IS_SAFARI && TOUCH_ENABLED` also will include iPhone as well?

Reply:
That is a good question.

Reply:
Yup, looks like we should add a `!! IS_IPHONE` or something.

Reply:
It could be out of scope, sorry for mixing two things. But, is it necessary to use regexp here to detect substring? Could it better for performance to use `includes` or `indexOf`? If you agree, i could make a PR with refactor of this utlis `browser.js` module.

Reply:
PR here https://github.com/videojs/video.js/pull/6371

Reply:
Unfortunately, `includes` isn't available on all platforms that we support and I'm not sure that the extra code that would arise from using `indexOf` is worth it, especially since it may end up being slower. Is there a specific benefit to using `include` or `indexOf`?

Reply:
The idea was to avoid regex because it much more performance demanding while `indexOf` or `include` could be more optimized (now or in later js engines). Although, it's not a huge optimization anyway.

Reply:
While I don't think benchmarks are that useful, this one does show that the behavior is different depending on the browser https://jsperf.com/startswith-vs-indexof-vs-regex/4. For example in Chrome `indexOf` is much faster while on Firefox `startsWith` is faster with the `Regexp#test` being on par. Also, given that this runs once, I'm not sure it really matters what is being done and shouldn't ever be in the critical path, so I'd be inclined to just leave it as is.

Reply:
Sounds reasonable! Thanks for the shared info. For me it also performed differently for mobile and for desktop with that benchmark http://jsben.ch/d4o38